functions/__tests__/functions.create.test.ts

```diff
@@ -1,10 +1,8 @@
 import algoliasearch from 'algoliasearch';
-import { firestore } from 'firebase-admin/lib/firestore';
 import * as functionsTestInit from 'firebase-functions-test';
 import mockedEnv from 'mocked-env';
 import { mocked } from 'ts-jest/utils';
 import { mockConsoleInfo } from './__mocks__/console';
-import {version} from '../src/version';
```

Comment:
some of the changes seem unrelated

Reply:
i had to refactor the tests to remove false positives.